backend/background/sources/collective_bargaining.py

```
@@ -13,196 +13,267 @@
-                for a_tag in a_tags:
-                    href = cast(Optional[str], a_tag.get("href"))
-                    if href and isinstance(href, str) and href.endswith(".pdf"):
-                        title: str = href.split("/")[-1].replace(".pdf", "")
+            campus_name = union_name
+            logger.info(f"Processing local campus: {campus_name}")
+
+            # Click to expand the accordion if it's not already expanded
+            try:
+                await button.click()
+                await page.wait_for_timeout(500)  # Wait for expansion
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)